sourceCode.py

```diff
@@ -1,26 +1,120 @@
+# example: curse with jordan, hold small-talk with jordan
+node_c = pydot.Node("Initiate Example Topic \n\
+                    Output 'Small-talk with Jordan'", style="filled", fillcolor="green", shape = 'box')
+node_c1 = pydot.Node("Initiate Example Topic \n\
+                     Output 'Cursing with Jordan'", style="filled", fillcolor="green", shape = 'box')
+node_c2 = pydot.Node("Initiate Example Topic \n\
+                     Output 'Complain about quarantine'", style="filled", fillcolor="green", shape = 'box')
+# example: Ok, you chose Example Topic. Why dont you start by telling me why you chose this topic?
+node_d = pydot.Node("Maintain Conversation \n\
+                    Output 'Tell me more.'", style="filled", fillcolor="orange", shape = 'box')
+node_d1 = pydot.Node("Maintain Conversation \n\
+                    Output 'Tell me more.'", style="filled", fillcolor="orange", shape = 'box')
+node_d2 = pydot.Node("Maintain Conversation \n\
+                    Output 'Tell me more.'", style="filled", fillcolor="orange", shape = 'box')
+# example: Open questions and fillers in that class of that specific example topic - Tell me more! How did that happen? I understand. 
+node_e = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+node_e1 = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+node_e2 = pydot.Node("End Conversation \n\
+                    Output 'Thank you for trusting me. Do you want to pick another topic or do you want to say goodbye?'", style="filled", fillcolor="yellow", shape = 'box')
+# example: Ok, you wanted to end this conversation. Do you want to choose another topic?
+
+
+# creating nodes for participants response
+
+# add nodes
+graph.add_node(node_a)
+graph.add_node(node_b)
+graph.add_node(node_c)
+graph.add_node(node_c1)
+graph.add_node(node_c2)
+graph.add_node(node_d)
+graph.add_node(node_d1)
+graph.add_node(node_d2)
+graph.add_node(node_e)
+graph.add_node(node_e1)
+graph.add_node(node_e2)
+
+
+
+# create edges
+graph.add_edge(pydot.Edge(node_a, node_b))
+graph.add_edge(pydot.Edge(node_b, node_c))
+graph.add_edge(pydot.Edge(node_b, node_c1))
+graph.add_edge(pydot.Edge(node_b, node_c2))
+
+graph.add_edge(pydot.Edge(node_c, node_d))
+graph.add_edge(pydot.Edge(node_c, node_e))
+graph.add_edge(pydot.Edge(node_c1, node_d1))
+graph.add_edge(pydot.Edge(node_c1, node_e1))
+graph.add_edge(pydot.Edge(node_c2, node_d2))
+graph.add_edge(pydot.Edge(node_c2, node_e2))
+graph.add_edge(pydot.Edge(node_e, node_b, label="alternatively go back to picking a topic", labelfontcolor="#009933", fontsize="10.0", color="blue"))
```

Comment:
Hi Marla! First of all, I really like your idea - it seems like a cool (and useful and ambitious) project! 

If I understand it correctly, you basically create three nodes (node_d, node_d1, node_d2, node_e, node_e1, node_e2) that are exactly the same, and then you combine them when creating the edges. Instead of writing this all out, you could use a for loop for it. I don't think this is necessary now but perhaps for future chunks of code.